### ISSUE TITLE
Mouse does work perfectly fine on tty when used on a system that has antix on it.

### ISSUE THREAD

**Issue by agoodfellow123** [cosmos72/twin#60]
The tutorial says:
> only xterm and derivatives have a standard to report mouse, but only report click/release, not drags

I did manage to get it working both clicking and dragging, what i did was:
Open twin by first booting my vm then without logging into the dm that antix has, i just changed to tty1 and opened twin. Everything worked perfectly. Probably about some process antix run at boot, you may want to look into that.

**Comment by klamonte** [comment]
ncurses mouse can support clicking and dragging, you just have to maintain knowledge of the button state.

**Comment by cosmos72** [comment]
My fault,
xterm documentation https://www.xfree86.org/current/ctlseqs.html#Mouse%20Tracking
states that it **can** also report mouse button release, "mouse highlight" and "button-motion events".

I will try to test them.

**Comment by tallship** [comment]
On 3/11/2021 1:07 AM, Massimiliano Ghilardi wrote:
> My fault,
> xterm documentation
> https://www.xfree86.org/current/ctlseqs.html#Mouse%20Tracking
> states that it *can* also report mouse button release, "mouse highlight"
> and "button-motion events".
> 
> I will try to test them.
> 

I've never had any issues on Slackware using gpm. In a Linux console or
an X session.


-- 
Bradley D. Thornton
Manager Network Services
http://NorthTech.US
TEL: +1.310.421.8268

**Comment by tallship** [comment]
On 3/11/2021 7:34 AM, Massimiliano Ghilardi wrote:
> Hi @tallship <https://github.com/tallship>,
> did you also try to run twin *inside* an xterm or similar,
> with |twstart --hw=tty| or equivalently |twin --hw=tty| from xterm shell
> prompt ?
> 
> That's what I just (re)tested, and it works better than what I remembered.
> 

Yes the twstart wasn't documented (lost rather, I discoverd it buried in
an obscure place at Sourceforge) for a **very** long time, and is
definitely the preferred way, then I just create several tiled twterms
each running a tmux session inside of them. so I can attach and detach
from twin, go home, and attach.

Everything is in one convenient place. Cut and paste is a little unusual.

Kinda what I wrote several years ago at:
http://northtech.us/content/20110606/throwing-away-your-password-enable-secure-logins-remote-hosts

I need to update that on the new site. I believe at the time that I had
built and hosted most of a website for you for the Twin project, but
life as a single parent trumped twin (that sounds like a pundit, but
it's not).

Now we have Github, which is much better, and I maintain a couple of
mirrors of it on my Gitea servers.

I also need to address the distro packages I maintain, they're woefully
out of date and yet, still work. A testament to how bulletproof Twin
actually always has been.

One of these days I may actually ask you for an old DOS port you might
have in mothballs so I can run it under Quarterdeck's DESQview/X on a
486DX. After 30 years I'm still 1:102/17 on FidoNet.

I was surprised that many of my GoDaddy associates actually took to it
like butter on bread when I worked there. It really is a game changer
that can just sit there, waiting for someone to attach to it for a view
into a myriad of machines in a pinch.

Max, the force was definitely with you when you created Twin. although
I've never use it beyond it's X-term (Twinterm) capabilities.



-- 
Bradley D. Thornton
Manager Network Services
http://NorthTech.US
TEL: +1.310.421.8268
